changelog/unreleased/issue-3116

```diff
@@ -0,0 +1,9 @@
+Bugfix: Error handling in restore
+
+Restic restore used to not print errors when getting file contents from
+the repository. It also didn't return a non-zero error code if there were
```

Comment:
That are a bit too many negations in my opinion. What about something like the following:
```
restore also incorrectly exited with a zero error code even when there were
errors during the restore process. Now, a non-zero exit code is returned.
```

Reply:
changed.